documentation/openapi-authoring-automated-guidelines.md

```diff
@@ -28,97 +28,101 @@ We request OpenAPI(Swagger) spec authoring be assigned to engineers who have an
-| [R2001](#R2001)	| [AvoidNestedProperties](#R2001) |
-| [R4002](#R4002)	| [LocationMustHaveXmsMutability](#R4002) |
-| [R2066](#R2066)	| [PostOperationIdContainsUrlVerb](#R2066) |
-| [R2015](#R2015) | [ParameterNotDefinedInGlobalParameters](#R2015) |
-| [R1010](#R1010) | [AvoidMSDNReferences](#R1010) |
-| [R2017](#R2017)	| [PutRequestResponseScheme](#R2017) |
-| [R1009](#R1009) | [DeleteInOperationName](#R1009) |
-| [R1005](#R1005) | [GetInOperationName](#R1005) |
-| [R1003](#R1003) | [ListInOperationName](#R1003) |
-| [R1006](#R1006)	| [PutInOperationName](#R1006) |
-| [R1007](#R1007) | [PatchInOperationName](#R1007) |
-| [R1011](#R1011) | [HttpsSupportedScheme](#R1011) |
-| [R2065](#R2065) | [LicenseHeaderMustNotBeSpecified](#R2065) |
-| [R2018](#R2018) | [XmsEnumValidation](#R2018) |
-| [R3060](#R3060) | [XmsPageableListByRGAndSubscriptions](#R3060) |
-| [R2063](#R2063) | [OperationIdNounConflictingModelNames](#R2063) |
-| [R2064](#R2064) | [LROStatusCodesReturnTypeSchema](#R2064) |
-| [R2023](#R2023) | [SummaryAndDescriptionMustNotBeSame](#R2023) |
+| Id | Rule Name | Applies to |
+| --- | --- | --- |
+| [R4000](#R4000-2) | [ParameterDescriptionRequired](#R4000-2) | ARM and Data plane Swagger specs |
+| [R4000](#R4000-3) | [DescriptiveDescriptionRequired](#R4000-3) | ARM and Data plane Swagger specs |
+| [R4000](#R4000-4) | [DescriptionAndTitleMissing](#R4000-4) | ARM and Data plane Swagger specs |
+| [R4000](#R4000-5) | [OperationDescriptionOrSummaryRequired](#R4000-5)  | ARM and Data plane Swagger specs |
+| [R2001](#R2001)	| [AvoidNestedProperties](#R2001) | ARM and Data plane Swagger specs |
+| [R4002](#R4002)	| [LocationMustHaveXmsMutability](#R4002) | ARM Swagger specs |
+| [R2066](#R2066)	| [PostOperationIdContainsUrlVerb](#R2066) | ARM and Data plane Swagger specs |
+| [R2015](#R2015) | [ParameterNotDefinedInGlobalParameters](#R2015) | ARM and Data plane Swagger specs |
+| [R1010](#R1010) | [AvoidMSDNReferences](#R1010) | ARM and Data plane Swagger specs |
+| [R2017](#R2017)	| [PutRequestResponseScheme](#R2017) | ARM and Data plane Swagger specs |
+| [R1009](#R1009) | [DeleteInOperationName](#R1009) | ARM and Data plane Swagger specs |
+| [R1005](#R1005) | [GetInOperationName](#R1005) | ARM and Data plane Swagger specs |
+| [R1003](#R1003) | [ListInOperationName](#R1003) | ARM and Data plane Swagger specs |
+| [R1006](#R1006)	| [PutInOperationName](#R1006) | ARM and Data plane Swagger specs |
+| [R1007](#R1007) | [PatchInOperationName](#R1007) | ARM and Data plane Swagger specs |
+| [R1011](#R1011) | [HttpsSupportedScheme](#R1011) | ARM Swagger specs |
+| [R2065](#R2065) | [LicenseHeaderMustNotBeSpecified](#R2065) | ARM and Data plane Swagger specs |
+| [R2018](#R2018) | [XmsEnumValidation](#R2018) | ARM and Data plane Swagger specs |
+| [R3060](#R3060) | [XmsPageableListByRGAndSubscriptions](#R3060) | ARM Swagger specs |
+| [R2063](#R2063) | [OperationIdNounConflictingModelNames](#R2063) | ARM and Data plane Swagger specs |
```

Comment:
Hmm... it was scheduled to be removed guess we haven't gotten there yet, you can disregard this for now.